index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div>
+      <header>
+        <div class="head">
+          <p class="log">My Portfolio</p>
+          <img
+            class="hum"
+            src="photo/Icon - Menu.png"
+            alt=""
+          />
+        </div>
+      </header>
+      <div class="main">
+        <h2>
+           I<span>&#39;</span>m Khaled Glad to see you! 
+        </h2>
+        <p class="Main-Text">
+          I<span>&#39;</span>m a software developer! I can help you build a
+          product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need
+          coded, don<span>&#39;</span>t hestiate to contact me.
+        </p>
+        <div class="Link">
+          <a class="link-Facebook" href="https://www.facebook.com/khaled.alaa.1420354/"
+            >let<span>&#39;</span>s connect</a
```

Comment:
- [ ] `lets connect` is not a link but a heading to the social icons. please fix that